benches/main.rs

```diff
@@ -201,3 +201,50 @@ fn x_combined(bench: &mut Bencher) {
         black_box(Time::parse_str(&t4).unwrap());
     })
 }
+
+
+#[bench]
+fn bench_format_date(bench: &mut Bencher) {
+    let date = Date{
+        year: 2022,
+        month: 7,
+        day: 10
+    };
+    bench.iter(|| {
+        date.to_string();
```

Comment:
You been to use black_box here, probably for both the input and the result.

Reply:
Yes, I have added this code